e2e/test/scenarios/organization/timelines-question.cy.spec.js

```diff
@@ -338,7 +338,7 @@ describe("scenarios > organization > timelines > question", () => {
         cy.findByLabelText("Event name").type("RC2");
         cy.findByLabelText("Date").clear().type("10/20/2023");
         cy.button("Create").click();
-        cy.wait("@createEvent");
+        waitForTimelinesAfterCreatingAnEvent("RC2");
```

Comment:
It actually failed here too, if you look at the [stress test result.](https://github.com/metabase/metabase/actions/runs/18773103938/job/53561476399#step:10:111)